internal/plugins/sdk/init.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: rewrite this when plugins phase 2 is implemented.
+package sdk
+
+import (
+	"fmt"
+	"io/ioutil"
+)
+
+// RunInit perform the SDK plugin-specific scaffolds.
+func RunInit(isGo bool) error {
```

Comment:
I would much rather see this as
```suggestion
func RunInit(cfg *config.Config) error {
```
and compare plugin names using `PluginKeyToProjectType`.